digitalocean/droplet/resource_droplet.go

```diff
@@ -141,6 +146,35 @@ func ResourceDigitalOceanDroplet() *schema.Resource {
 				Default:  false,
 			},
 
+			"backup_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
```suggestion
				Optional: true,
				RequiredWith: []string{"backups"},
```

Since we're requiring that `backups` is set when using `backup_policy`, we probably want to use `RequiredWith` here too in order to give early feedback.

https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema#Schema.RequiredWith